build.sbt

```diff
@@ -86,62 +60,60 @@ lazy val kernel =
       name := "trace4cats-kernel",
       libraryDependencies ++= Dependencies.test.map(_ % Test),
       buildInfoKeys := Seq[BuildInfoKey](version, scalaVersion, sbtVersion),
-      buildInfoPackage := "io.janstenpickle.trace4cats.kernel"
+      buildInfoPackage := "io.janstenpickle.trace4cats.kernel",
+      Test / unmanagedSourceDirectories ++= Seq(
+        baseDirectory.value / ".." / "testkit" / "src" / "main" / "scala",
+        baseDirectory.value / ".." / "core" / "src" / "main" / "scala"
+      ),
+      libraryDependencies ++= Seq(
+        Dependencies.catsEffectStd,
+        Dependencies.commonsCodec,
+        // Dependencies.kittens, // TODO re-add once compatible with Scala 3
+        Dependencies.caseInsensitive,
+        Dependencies.collectionCompat,
+      ),
+      libraryDependencies ++= (Dependencies.test ++ Seq(
+        Dependencies.fs2,
+        Dependencies.log4cats,
+        Dependencies.hotswapRef
+      )).map(_ % Test)
     )
-    .dependsOn(model, testkit % Test)
+    .dependsOn(`fp-utils` % Test)
     .enablePlugins(BuildInfoPlugin)
 
 lazy val core =
   (project in file("modules/core"))
     .settings(publishSettings)
     .settings(
       name := "trace4cats-core",
-      libraryDependencies ++= Seq(Dependencies.collectionCompat),
+      libraryDependencies ++= Seq(
+        Dependencies.collectionCompat,
+        Dependencies.fs2,
+        Dependencies.log4cats,
+        Dependencies.hotswapRef
+      ),
       libraryDependencies ++= Dependencies.test.map(_ % Test)
     )
-    .dependsOn(kernel, testkit % Test, `exporter-common` % Test)
+    .dependsOn(kernel, `fp-utils`, testkit % Test)
 
-lazy val base =
-  (project in file("modules/base"))
+lazy val `fp-utils` =
```

Comment:
This triggers me the most :) The first version of tofu was named `fp-utils` which was recognized by its authors as a bad name later.

Can we find a more clear name related to contextual effects?

Reply:
> Do we have a written plan on package renaming? Currently, the new module organization looks inconsistent with packages and triggers my OCD.

Not yet, I wanted to get the modules moved about first. I'll create an issue for us to discuss.

Reply:
I'm open to other names, this was just suggested in the original issue. Maybe just `context-utils` or `contextual-utils` is ok? I realise it still contains "utils", but unlike tofu it's a utilities library for t4c, not a standalone project.

Reply:
I'm fine with `context`